### PR TITLE
feat(cmd): snapshots command

### PR DESCRIPTION
**Pull request by rootulp**
Closes https://github.com/celestiaorg/celestia-app/issues/2824

## Testing

I verified the snapshots command was added

```shell
$ ./build/celestia-appd snapshots
Manage local snapshots

Usage:
  celestia-appd snapshots [command]

Available Commands:
  delete      Delete a local snapshot
  dump        Dump the snapshot as portable archive format
  export      Export app state to snapshot store
  list        List local snapshots
  load        Load a snapshot archive file (.tar.gz) into snapshot store
  restore     Restore app state from local snapshot

Flags:
  -h, --help   help for snapshots

```

### CODE REVIEW

File: cmd/celestia-appd/cmd/root.go [hunk 1]
```diff
@@ -149,6 +150,9 @@ func initRootCmd(rootCmd *cobra.Command, encodingConfig encoding.Config) {
 		keys.Commands(app.DefaultNodeHome),
 		bscmd.VerifyCmd(),
 	)
+	rootCmd.AddCommand(
+		snapshot.Cmd(NewAppServer),
+	)
```
Comment (cmwaters) [inline cmd/celestia-appd/cmd/root.go:155]: nit: I guess this could have just been appended to the `AddCommand` function above

**Review by cmwaters — APPROVED** [review]
ty! this turned out to be easier than I thought. Can we backport this to v1.x?

File: cmd/celestia-appd/cmd/root.go (hunk 1 above)
Comment (rootulp) [reply cmd/celestia-appd/cmd/root.go:155]: true. TBH the contents of this function don't make sense to me. IDK why we add some commands in a block at the top and then add more later

Comment (cmwaters) [reply cmd/celestia-appd/cmd/root.go:155]: yeah it's all a bit messy but not too problematic

**Pull request merged**